### ISSUE TITLE
How to compute the gradient of reward w.r.t. torso mass?

### ISSUE THREAD

**Issue by ZJP-c4cld** [deepmind/dm_control#300]
I want to construct a neural network in pytorch which outputs appropriate values of mujoco model parameters. But this requires that mujoco model parameters are converted into pytorch tensors , and mujoco environments run well with these converted model parameters, and compute the gradient of reward w.r.t the mujoco model parameters.
Anyone know how to achieve this goal?
